app/config/local.json

```diff
@@ -0,0 +1,16 @@
+{
+  "ORIGIN": "localhost",
+  "PORT": 3000,
+  "PGUSER": "ccbc_app",
+  "PGPASSWORD": "",
+  "PGHOST": "localhost",
+  "PGPORT": 5432,
+  "PGDATABASE": "ccbc",
+  "PGSCHEMA": "ccbc_public",
+  "HOST": "localhost",
+  "OPENSHIFT_APP_NAMESPACE": "ff61fb-dev",
+  "SESSION_SECRET": "superSecretSession",
+  "SITEMINDER_LOGOUT_URL": "https://logontest7.gov.bc.ca/clp-cgi/logoff.cgi",
+  "ENABLE_MOCK_TIME" : true,
```

Comment:
Is this what is leading to happo displaying the time travel components? Should we set it to `false`, so that happo displays screenshots that match the production site?
`ENABLE_MOCK_COOKIES` below should remain `true` to allow cypress to mock the current date

Reply:
Done

Reply:
Now happo screenshots do not have Time Travel